pages/common/rustc.md

```diff
@@ -0,0 +1,19 @@
+# rustc
+
+> The Rust compiler.
+
+- Compile a single file:
+
+`rustc {{file.rs}}`
+
+- Compile with high optimization:
+
+`rustc -O {{file.rs}}`
+
+- Compile with debugging information:
+
+`rustc -g {{file.rs}}`
+
+- Build a test harness:
```

Comment:
I was digging a bit more about it and the `--test` option is currently very poorly documented and is almost of none practical use for the user because **cargo** manages the testing. So If it is not practical i think it doesn't belong in the *tldr page*.